src/pages/contributing/component/index.mdx

```diff
@@ -521,19 +522,211 @@ leaving any other comments on
+|         | Disabled | border-top | `$border-subtle` |
+| Title   | Disabled | background | `$text-disabled` |
+| Icon    | Disabled | fill       | `$icon-disabled` |
+
+<Row>
+<Column colLg={12}>
+
+<Tabs>
+
+<Tab label="Default">
+
+![default accordion alignment interactive states](images/accordion-style-2.png)
+
+</Tab>
+
+<Tab label="Flush">
+
+![flush accordion alignment interactive states](images/accordion-style-3.png)
+
+</Tab>
+
+</Tabs>
+
+</Column>
+</Row>
+
+## Typography
+
+<!--
+Begin this section with any brief content guidelines. Link out to additional guidelines if needed.
+
+Then, use a table to document font-size, font-weight, and type tokens in this section. You can reference this [table generator](https://www.tablesgenerator.com/markdown_tables) for easy table creation.
+
+The following table shows an example from [Accordion](https://carbondesignsystem.com/components/accordion/style).
+-->
+
+All accordion titles are set in sentence case. See the accordion
+[content guidelines](/components/accordion/usage#content) for more details.
+
+| Element | Font-size (px/rem) | Font-weight   | Type token |
+| ------- | ------------------ | ------------- | ---------- |
+| Title   | 14 / 0.875         | Regular / 400 | `$body-01` |
+| Content | 14 / 0.875         | Regular / 400 | `$body-01` |
+
+## Structure
+
+<!--
+Use a table to document states, pacing, and any other component-specific configurations. You can reference this [table generator](https://www.tablesgenerator.com/markdown_tables) for easy table creation.
+
+Add supporting images to this section, with at least one image showing how color is applied within the component and another showing structure and spacing measurements. Images are accompanied by a descriptive caption below the image and used in conjunction with the spec charts. Note we used fixed images in the Structure section.
+
+The following table shows an example from [Accordion](https://carbondesignsystem.com/components/accordion/style).
+-->
+
+| Element | Property    | px/rem   | Spacing token |
+| ------- | ----------- | -------- | ------------- |
+| Header  | height      | 40 / 2.5 | –             |
+| Item    | border-top  | 1        | –             |
+| Title   | margin-left | 16 / 1   | `$spacing-05` |
+
+<div className="image--fixed">
+
+![Structure measurements for default accordion alignment](images/accordion-style-4.png)
+
+</div>
+
+<Caption>
+  Structure measurements for default accordion alignment. | px / rem
+</Caption>
+
+<br />
+
+<div className="image--fixed">
+
+![Spacing measurements for default accordion alignment](images/accordion-style-5.png)
+
+</div>
+
+<Caption>
+  Spacing measurements for default accordion alignment. | px / rem
+</Caption>
+
+### Variant structure
+
+<!--
+You main need to provide additional specs for component variants or default versus flush alignment. For each variant name a new section and give it a name. For example: Default alignment structure.
```

Comment:
oh wait I see, we're naming each variant and not doing a general one under the H2 structure. Maybe explain that in the note here. "If a component has variants, instead of having general section (like mentioned above) under the H2, you'll want to break up the structure section by variant" but worded better.

Reply:
Yes, after our meeting last week I spent a little time refining this wording and describing how the Accordion example has no variants and therefore no additional headers.